Cesium.CodeGen/Contexts/IDeclarationScope.cs

```diff
@@ -45,6 +46,10 @@ internal interface IDeclarationScope
     /// </summary>
     /// <returns>Name of the virtual label which can be used by continue statement</returns>
     string? GetContinueLabel();
+    
+    void PushSpecialEffect(object declaration);
```

Comment:
Please add a comment indicating our pragma as an example of this "special effect", for future readers.